track_and_trace/asset_client/src/services/parsing.js

```diff
@@ -18,7 +18,8 @@
 
 const _ = require('lodash')
 const moment = require('moment')
-const { FLOAT_PRECISION } = require('./payloads')
+
+const FLOAT_PRECISION = 1000000
```

Comment:
There are a lot of deletions in this commit, but I'm unsure based on the commit message as to why

Reply:
updated commit so changes updating the deleted code are part of this commit now.